### PR TITLE
Standardized to the pfappserver::Controller::Config

### PR DESCRIPTION
**Pull request by jrouzierinverse**

### CODE REVIEW

**Comment by jrouzierinverse** [comment]
Please review

**Comment by julsemaan** [comment]
I guess I'll test tomorrow more than review since it's mostly search and replace.

File: html/pfappserver/lib/pfappserver/Controller/Config/Profile.pm [hunk 1]
```diff
@@ -1,8 +1,8 @@
-package pfappserver::Controller::Portal::Profile;
+package pfappserver::Controller::Config::Profile;
 
 =head1 NAME
 
-pfappserver::Controller::PortalProfile add documentation
+pfappserver::Controller::Config::Profile add documentation
```
Comment (julsemaan) [inline html/pfappserver/lib/pfappserver/Controller/Config/Profile.pm:5]: let's kill the add documentation virus

File: html/pfappserver/lib/pfappserver/Controller/Config/Profile/Default.pm [hunk 2]
```diff
@@ -1,8 +1,8 @@
-package pfappserver::Controller::Portal::Profile::Default;
+package pfappserver::Controller::Config::Profile::Default;
 
 =head1 NAME
 
-pfappserver::Controller::PortalProfile add documentation
+pfappserver::Controller::ConfigProfile add documentation
```
Comment (julsemaan) [inline html/pfappserver/lib/pfappserver/Controller/Config/Profile/Default.pm:5]: remove documentation

File: html/pfappserver/lib/pfappserver/Form/Config/Authentication/Source/Blackhole.pm [hunk 3]
```diff
@@ -1,21 +1,21 @@
-package pfappserver::Form::Authentication::Source::Blackhole;
+package pfappserver::Form::Config::Authentication::Source::Blackhole;
 
 =head1 NAME
 
-pfappserver::Form::Authentication::Source::Blackhole add documentation
+pfappserver::Form::Config::Authentication::Source::Blackhole add documentation
```
Comment (julsemaan) [inline html/pfappserver/lib/pfappserver/Form/Config/Authentication/Source/Blackhole.pm:5]: remove documentation

File: html/pfappserver/lib/pfappserver/Form/Config/Authentication/Source/Null.pm [hunk 4]
```diff
@@ -1,21 +1,21 @@
-package pfappserver::Form::Authentication::Source::Null;
+package pfappserver::Form::Config::Authentication::Source::Null;
 
 =head1 NAME
 
-pfappserver::Form::Authentication::Source::Null add documentation
+pfappserver::Form::Config::Authentication::Source::Null add documentation
```
Comment (julsemaan) [inline html/pfappserver/lib/pfappserver/Form/Config/Authentication/Source/Null.pm:5]: here too

File: html/pfappserver/lib/pfappserver/Form/Config/Authentication/Source/SponsorEmail.pm [hunk 5]
```diff
@@ -1,8 +1,8 @@
-package pfappserver::Form::Authentication::Source::SponsorEmail;
+package pfappserver::Form::Config::Authentication::Source::SponsorEmail;
 
 =head1 NAME
 
-pfappserver::Form::Authentication::Source::SponsorEmail - Web form for email-based self-registration by soonsor
+pfappserver::Form::Config::Authentication::Source::SponsorEmail - Web form for email-based self-registration by soonsor
```
Comment (julsemaan) [inline html/pfappserver/lib/pfappserver/Form/Config/Authentication/Source/SponsorEmail.pm:5]: *self-registration by sponsor

**Comment by dwlfrth** [comment]
Unfortunately, as @julsemaan said, there not much to review...
We'll have to extensively test it to make sure nothing is broken.

One thing that could help us testing it... Can you add the impacts of the changes in the pull-request documentation (per example: Affect administration GUI only, administration and configurator GUI, ....)

**Comment by jrouzierinverse** [comment]
This only affects the following in the Admin GUI

Portal Profiles

Admin Access

Switches

WRIX

Sources

Firewall SSO

Floating Devices

Fingerprints

User Agents

MAC Addresses

**Comment by jrouzierinverse** [comment]
Please review

**Pull request merged**